MAUI/PDF-Viewer/Custom-Bookmark.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Custom Bookmark in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here about the custom bookmark navigation in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Custom bookmark in .NET MAUI PDF Viewer
+
+A PDF document may optionally have custom bookmarks that allow the user to bookmark pages and navigate to them. The PDF viewer control displays these custom bookmarks in outline view.
+
+## Showing/hiding the custom bookmark
+
+The PDF viewer's built-in custom bookmark view, which displays the custom bookmarks, can be shown or hidden using the [IsOutlineViewVisible](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_IsOutlineViewVisible) property. Its default value is `false`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<pdfViewer:SfPdfViewer x:Name="pdfViewer" IsOutlineViewVisible="{Binding OutlineViewVisible}" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+pdfViewer.IsOutlineViewVisible = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Accessing custom bookmarks collection
+
+To access the custom bookmarks, you can use the `CustomBookmarks` collection. This property provides the list of custom bookmarks in the PDF. 
+
+{% tabs %}
+{% highlight c# %}
+
+var customBookmarks = pdfViewer.CustomBookmarks;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add , Edit and Remove custom bookmark
+
+To add, edit, or remove custom bookmarks in a PDF viewer, you can use the `CustomBookmarks` collection. This property provides access to the collection of custom bookmarks that can be manipulated to modify the bookmarks displayed in the viewer's bookmark view.
+
+### Add custom bookmarks using the bookmark pane
+
+Custom bookmarks can be added using the floating button in the bookmark pane.
+
+![Custom bookmark floating button](Images\custom-bookmark.png)
+
+### Adding a Custom Bookmark programmatically
+
+To add a custom bookmark, you can create a new instance of Bookmark and add it to the `CustomBookmarks` collection:
+
+{% tabs %}
+{% highlight c# %}
+
+Bookmark newBookmark = new Bookmark()
+{
+    Name = "New Bookmark",
+    PageNumber = 1, // Page number where the bookmark should to navigate
+};
+
+pdfViewer.CustomBookmarks.Add(newBookmark);
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Rename custom bookmarks using the bookmark context menu
+
+Tap the context menu button on the custom bookmark to be edited and choose Rename and enter the desired name.
```

Comment:
Tap the context menu button on the custom bookmark to be edited and then choose Rename and enter the desired name.